go.graphics.swing/src/go/graphics/swing/event/swingInterpreter/GOSwingEventConverter.java

```diff
@@ -45,6 +49,8 @@
 	 */
 	private boolean panWithButton3;
 
+	private static Window referenceWindowToCalculateRetinaScale;
+
```

Comment:
It's always a good idea to try not to create static fields, as this creates a state in the game, which cannot easily be reset. Instead of setting the Window to this field, you could get the GraphicsConfiguration from the Component given to the constructor.

Therefore I think it's best to calculate the scale factor only once in the constructor and save the factor in a (non-static) field. This also ensures that no other class has to know about this implementation detail of this class. Especially SwingManagedJSettlers does not have to be modified / even know about this class.

Can you do the changes? Besides that: Thanks for your contribution!